db-manager.js

```diff
@@ -57,15 +57,15 @@ module.exports.loginUser = async function(email, password) {
   try {
     const user = await User.findOne({ Email: email }).exec();
     if (!user) {
-      throw "Invalid Email/Password combination";
+      throw "1 Email/Password combination";
     }
```

Comment:
I don't understand this change exactly, can you elaborate a bit? This string is shown to the end user and I don't understand why the "1" is there.